index.html

```diff
@@ -1 +1,39 @@
-task1-3
\ No newline at end of file
+<!DOCTYPE html>
+<html lang="ja">
+<head>
+    <meta charset="utf-8">
+    <title>task3</title>
+</head>
+<body>
+    <script>
+        
+        
+        
+        function task (){  
```

Comment:
```suggestion
        function task(){  
```

関数名と引数を記入するカッコは離さない方がいいかと思いました。